examples/dataset/object_detection_2d/object_detection_2d/constants.py

```diff
@@ -13,6 +13,7 @@
 # limitations under the License.
 BUCKET = "kolena-public-examples"
 DATASET = "coco-2014-val"
+WORKFLOW = "transportation"
```

Comment:
Should this be "object-detection-2d", in terms of breaking down by task type?

Reply:
Maybe "workflow" is the wrong term here. The folders are `image-retrieval-by-text`, `person-detection`, `transportation`, so "task" may be better suited

Reply:
Asking more about should this we match the folder to the example, i.e. "object_detection_2d" or does "transportation" more reflective on the problem statement?